cmd/internal/planetscale_edge_database.go

```diff
@@ -276,6 +303,7 @@ func (p PlanetScaleEdgeDatabase) sync(ctx context.Context, tc *psdbconnect.Table
 		TableName:  s.Name,
 		Cursor:     tc,
 		TabletType: tabletType,
+		Cells:      cells,
```

Comment:
@Phanatic Would it be better to change how `psdbconnect.SyncRequest` behaves if no `Cells` are given that it defaults to all cells and not only the current vtgate cell?